### PR TITLE
[cu-2nbhjbb] Fix Undefined label in chart

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
[the chart is displayed with predicted price calculation. However, the beginning fee is displayed as undefined.](https://app.clickup.com/t/2nbhjbb)

## Description
If there's an empty series at start chart shows an undefined, i've changed to only show one series at a time if there's no data in the other

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2nbhjbb Pablo - (AUCTION) LBP chart not working properly](https://app.clickup.com/t/2nbhjbb)

File: frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx [hunk 1]
```diff
@@ -7,6 +7,57 @@ import { MockedAsset } from "@/store/assets/assets.types";
...
+        </Typography>
+        <FiberManualRecordIcon color="inherit" />
+        <Typography variant="body2" pl={1} whiteSpace="nowrap">
+          {assetSymbol} predicted price (without new buyers)
+        </Typography>
+      </>
+    );
+  }
+
+  if (priceSeries.length <= 0 || predictedPriceSeries.length <= 0) {
```
Comment (easteregg) [inline frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx:36]: (question)

Why not use `priceSeries.length === 0` or the simpler: `Array.isArray(priceSeries) && !priceSeries.length`?

Comment (saadjhk) [reply frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx]: yes for sure, updated

File: frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx [hunk 2]
```diff
@@ -7,6 +7,57 @@ import { MockedAsset } from "@/store/assets/assets.types";
...
+          <FiberManualRecordIcon color="primary" />
+          <Typography variant="body2" pl={1} pr={2}>
+            {assetSymbol}
+          </Typography>
+        </>
+      );
+    }
+  }
+  return null;
+};
```
Comment (andresmechali) [inline frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx:59]:
```suggestion
  if (priceSeries.length > 0 && predictedPriceSeries.length > 0) {
    return null;
  }
  
  return (
    <>
      <FiberManualRecordIcon color="primary" />
      <Typography variant="body2" pl={1} pr={2}>
        `${assetSymbol} ${priceSeries.length === 0 ? "predicted price (without new buyers)" : ""}`
      </Typography>
    </>
  )
```

**Review by andresmechali — COMMENTED** [review]
Just a small suggestion

File: frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx (hunk 2 above)
Comment (saadjhk) [reply frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx:59]: @andresmechali `priceSeries` and `predictedPriceSeries` we have to show labels for both series, similarly, if data exists within one of the two we want to show only that label

**Review by andresmechali — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**